snuba/datasets/errors_replacer.py

```diff
@@ -259,6 +266,14 @@ def get_projects_query_flags(
         p.zremrangebyscore(type_key, float("-inf"), now - settings.REPLACER_KEY_TTL)
         p.zrevrangebyscore(type_key, float("inf"), now - settings.REPLACER_KEY_TTL)
 
+    for exclude_groups_key, _ in exclude_groups_keys_and_types:
+        p.zrevrange(
```

Comment:
This `zrevrange` gets the latest item only and has `withscores=True` so we can get its score. The other call is getting all items but without the scores. I could change it so that call also responds with the scores but it would be a lot of extra data we don't need since only the latest score matters.